docs/docs/v6-upgrade-guidance.md

```diff
@@ -168,6 +168,17 @@ As a side issue, we've also updated the response output of the `spo group member
 
 Update your scripts to use the new `member` noun instead of `user`. If you are using the output of `spo group member list` in JSON output mode, update your scripts and remove the `value` object. 
 
+## Logic to retrieve installed team apps with `teams app list` has changed
+The logic to list the installed apps in a specified team is moved to a new command `teams team app list`. As a result, the command `teams app list` only displays the installed apps from the Microsoft Teams app catalog. The command `teams app list` does no longer contains the options `all`, `teamId` and `teamName`. In addition, there is a new option for this command that allows you to indicate which installed apps from the Microsoft Teams app catalog you want to list according to the distribution method.
+
+The updated documentation of these commands
+- [teams app list](./cmd/teams/app/app-list.md)
+`- [teams team app list](./cmd/teams/team/app-list.md)`
```

Comment:
This is not valid Markdown syntax, could you have a look at it?
Bullet list should be preceded by an enter, also the back ticks have no use here.

Reply:
As said in my post, this is a link to the not yet existing command `teams team app list`, so it putted it in comment, otherwise the pr build fails

Reply:
Then we should maybe put this PR on hold until the command has been merged. Anyway, your bullet list is still broken as said.

![image](https://user-images.githubusercontent.com/11723921/203418855-c7a65a3d-e552-4104-9139-c4c5cc4f2a82.png)

Reply:
Thank you for the feedback Milan! I've updated the pr